dts/riscv/bflb/bl60x.dtsi

```diff
@@ -83,6 +123,22 @@
 			};
 		};
 
+		clocks: clock-controller@40000000 {
+			compatible = "bflb,bl60x-clock-controller", "bflb,clock-controller";
+			reg = <0x40000000 DT_SIZE_K(4)>;
+			#clock-cells = <1>;
+			status = "okay";
+			clocks = <&clk_rc32m>, <&clk_crystal>, <&clk_root>, <&clk_bclk>,
+				<&clk_pll BL60X_PLL_INVALID>,
```

Comment:
`<&clk_pll BL60X_PLL_INVALID>,`
I think you should only add valid options here.

Reply:
it's for coherency, so pll isnt accessed through 'pll_192' (for example) but through 'pll' in the code, is this necessary to remove?

Reply:
As far I know you should list the valid clock inputs.

Reply:
done